.sync.yml

```diff
@@ -1 +1,5 @@
 ---
+.travis.yml:
+  extras:
+  - rvm: 2.1.9
+    script: bundle exec rake rubocop
```

Comment:
Can you remove this line break?

Reply:
There wasn't a line break. I've added one as it seems that the github diff was highlighting this as an issue?

Reply:
Hmm strange, when I commented there was a small red arrow at the end of the line. Anyway looks good to me.

Reply:
No no, you're right, there was, adding a line break got rid of it :)